lib/graph/Collaboratory/User.js

```diff
@@ -88,16 +88,33 @@ COL_User.prototype.activities = function(query, callback) {
 
 
 /**
- * Retrieves a list user courses - published or not
+ * Retrieves a list users private clasess
```

Comment:
tiny typo here 😛

Reply:
Where is the typo?

Reply:
"Retrieves a list users private clasess" <-- don't worry all geniuese suffer from dyslexia 😛